corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -34,24 +34,27 @@
 MOBILE_UCR_RANDOM_THRESHOLD = 1000
 
 
-def _should_sync(restore_state):
+def _should_sync(restore_state, is_v2):
     last_sync_log = restore_state.last_sync_log
     if not last_sync_log or restore_state.overwrite_cache:
         return True
 
-    sync_interval = restore_state.restore_user.get_mobile_ucr_sync_interval()
-    if sync_interval is None and restore_state.params.app:
-        app = restore_state.params.app
-        if restore_state.params.app.copy_of:
-            # get sync interval from latest app version so that we don't have to deploy a new version
-            # to make changes to the sync interval
-            try:
-                app = get_brief_app(restore_state.domain, restore_state.params.app.copy_of)
-            except NoResultFound:
-                pass
-        sync_interval = app.mobile_ucr_sync_interval
-    if sync_interval is None:
-        sync_interval = restore_state.project.default_mobile_ucr_sync_interval
+    sync_interval = None
+    if not is_v2:
+        # in v2 sync delay is handled per report in the app
```

Comment:
I intentionally left this in case we wanted a hard minimum for an app, but I suppose multiple configurations could be confusing. Is that your thought for removing this?

Reply:
yea I think it's a bit confusing but I guess having a hard min could be useful. I'll think about it some more.

Reply:
Cool. I'm not against this change, so happy to merge once tests pass.

we could also use the `<input min>` attribute to specify a minimum on each input to make it less confusing, although we may want some warning that the minimum is due to an app /domain level setting

Reply:
Went back and forth on this for ages and decided to remove the user and app settings so that it's just the domain and individual report settings.

Nobody is using the user / app settings currently anyway.

Reply:
Still need to make it possible to ignore the sync delay for test users.